tables/Cop-Eg-Comp8.utb

```diff
@@ -0,0 +1,186 @@
+# liblouis: Multi Language Coptic Computer Braille table
+
+# Copyright (C) 2020 By Ibraam Nasif, St. Didimous Group  Coptic Bishopric of Youth - Egypt
+#
+# This file is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2.1 of the License, or (at your option) any later version.
+# This file is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+# You should have received a copy of the GNU Lesser General Public
+# License along with this file; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+
+    #-index-name: Coptic, Computer Braille 
+    #-display-name: Coptic Egypt computer braille
+    #-author-name: Ibraam Nasif 
+    #-author-email: ibraam.wahib@gmail.com
+    
+    #+locale: Coptic 
+    #+type: computer
+    #+dots: 8
+    #+direction: forward
```

Comment:
Wouldn't it be fairly simple to make it work both directions? It's computer braille after all. Is there a specific reason you put "forward" here?